src/main.rs

```diff
@@ -24,6 +29,15 @@ fn check_config(config: inputs::ConfigInput) -> failure::Fallible<()> {
     Ok(())
 }
 
+fn send_metrics(id: &identity::Identity) -> failure::Fallible<()> {
```

Comment:
We are trying not to call these "metrics", as it just creates confusion with what most of the people call "metrics", i.e. https://prometheus.io/docs/concepts/metric_types/.

Unless we are planning to actually collect and push only that kind of `(string, float)` records (is that the case?), let's try to find a better name for this.

Reply:
No, that's not the case. Will be renaming those bits and other parts with the name "metrics". Thanks for the clarification!

Reply:
Updated